python/cudf/cudf/core/dataframe.py

```diff
@@ -488,6 +484,51 @@ def __getitem__(self, arg):
             msg = "__getitem__ on type {!r} is not supported"
             raise TypeError(msg.format(type(arg)))
 
+    def where(self, cond, other):
+        """
+        Replace values with other where the condition is False.
+
+        Parameters
+        ----------
+        cond : boolean
+            Where cond is True, keep the original value. Where False,
+            replace with corresponding value from other.
+        other: list of scalars or a dataframe,
```

Comment:
This should also be able to be a scalar and we should note that callables are not yet supported